### PR TITLE
Update Dutch strings.po

### PR DESCRIPTION
**Pull request by mourix**
Added all missing strings for the Dutch language.

Also made spelling and meanings more consistent after testing the menu's with previous translations.

### CODE REVIEW

File: language/Dutch/strings.po [hunk 1]
```diff
@@ -165,7 +165,7 @@ msgstr "Selecteer"
 
 #: src/frontend/mame/ui/dirmenu.cpp:58
 msgid "GameOver"
-msgstr "Game over"
+msgstr "Gameover"
```
Comment (happppp) [inline language/Dutch/strings.po:168]: "Game over" was correct

File: language/Dutch/strings.po [hunk 2]
```diff
@@ -1002,11 +1002,11 @@ msgstr "Jaar"
 
 #: src/frontend/mame/ui/utils.cpp:75
 msgid "Save Supported"
-msgstr "Opslaan ondersteund"
+msgstr "Ondersteund opslaan"
```
Comment (happppp) [inline language/Dutch/strings.po:1006]: Ondersteunt

File: language/Dutch/strings.po [hunk 3]
```diff
@@ -1002,11 +1002,11 @@ msgstr "Jaar"
...
 
 #: src/frontend/mame/ui/utils.cpp:76
 msgid "Save Unsupported"
-msgstr "Opslaan niet ondersteund"
+msgstr "Ondersteund opslaan niet"
```
Comment (happppp) [inline language/Dutch/strings.po:1009]: Ondersteunt

File: language/Dutch/strings.po [hunk 4]
```diff
@@ -1255,7 +1255,7 @@ msgstr "Normale tekst"
 
 #: src/frontend/mame/ui/custui.cpp:452
 msgid "Selected color"
-msgstr "Geselecteerde kleur"
+msgstr "Geselecteerd kleur"
```
Comment (happppp) [inline language/Dutch/strings.po:1258]: was correct, kleur is een "de" woord

File: language/Dutch/strings.po [hunk 5]
```diff
@@ -1287,7 +1287,7 @@ msgstr "Dipswitch"
 
 #: src/frontend/mame/ui/custui.cpp:460
 msgid "Unavailable color"
-msgstr "Onbeschikbare kleur"
+msgstr "Onbeschikbaar kleur"
```
Comment (happppp) [inline language/Dutch/strings.po:1290]: was correct, kleur is een "de" woord

File: language/Dutch/strings.po [hunk 6]
```diff
@@ -1597,11 +1597,11 @@ msgstr "Geluid\tOK\n"
 
 #: src/frontend/mame/ui/selgame.cpp:1079
 msgid "Capture\tUnimplemented\n"
-msgstr ""
+msgstr "Opname\tOngeïmplementeerd\n"
```
Comment (happppp) [inline language/Dutch/strings.po:1600]: "Niet geïmplementeerd" is beter?

File: language/Dutch/strings.po [hunk 7]
```diff
@@ -2921,15 +2921,16 @@ msgstr ""
 
 #: plugins/cheatfind/init.lua:504
 msgid "Pause Mode"
-msgstr ""
+msgstr "Pauzeermodus"
 
 #: plugins/cheatfind/init.lua:511
 msgid "Manually pause & unpause the game when needed with the pause hotkey"
 msgstr ""
+"Handmatig het spel wanneer nodig pauzeren & hervatten met de pauze sneltoets"
```
Comment (happppp) [inline language/Dutch/strings.po:2929]: & = en?
(I'm unsure if usage of "&" is correct in English here either)

File: language/Dutch/strings.po [hunk 8]
```diff
@@ -2921,15 +2921,16 @@ msgstr ""
...
 
 #: plugins/cheatfind/init.lua:536
 msgid "All slots cleared and current state saved to Slot 1"
-msgstr ""
+msgstr "Alle slots geleegd en huidige state opgeslagen naar slot 1"
```
Comment (happppp) [inline language/Dutch/strings.po:2933]: naar = in

File: language/Dutch/strings.po (hunk 1 above)
Comment (mourix) [reply language/Dutch/strings.po:168]: strings.po contains seperate strings "Game Over" and "GameOver". Are you sure this ends up correctly?

File: language/Dutch/strings.po (hunk 2 above)
Comment (mourix) [reply language/Dutch/strings.po:1006]: Will fix

File: language/Dutch/strings.po (hunk 3 above)
Comment (mourix) [reply language/Dutch/strings.po:1009]: Will fix

File: language/Dutch/strings.po (hunk 4 above)
Comment (mourix) [reply language/Dutch/strings.po:1258]: This string is used in the ui color menu. It means the color for "selected" items in the ui. So "geselecteerde kleur" did not make sense. Do you have a better translation?

File: language/Dutch/strings.po (hunk 5 above)
Comment (mourix) [reply language/Dutch/strings.po:1290]: Same as previous. It's the color of items marked as unavailable in the UI. In the menu it makes more sense..

File: language/Dutch/strings.po (hunk 6 above)
Comment (mourix) [reply language/Dutch/strings.po:1600]: This is a bit of a gray area as it's imperfect dutch, but a lot closer to the original string. Shall I change them all or not?

File: language/Dutch/strings.po (hunk 7 above)
Comment (mourix) [reply language/Dutch/strings.po:2929]: Indeed, I translated the original string. But I'll fix it now

File: language/Dutch/strings.po (hunk 8 above)
Comment (mourix) [reply language/Dutch/strings.po:2933]: Will fix

File: language/Dutch/strings.po (hunk 1 above)
Comment (happppp) [reply language/Dutch/strings.po:168]: I don't know, ok leave it at Gameover.

File: language/Dutch/strings.po (hunk 6 above)
Comment (happppp) [reply language/Dutch/strings.po:1600]: Yeah, change them

File: language/Dutch/strings.po (hunk 4 above)
Comment (happppp) [reply language/Dutch/strings.po:1258]: Ok understood, it's fine then

File: language/Dutch/strings.po [hunk 9]
```diff
@@ -56,7 +56,7 @@ msgstr "Bezels"
 
 #: src/frontend/mame/ui/videoopt.cpp:220
 msgid "CPanels"
-msgstr "BPanelen"
+msgstr "CPanelen"
```
Comment (happppp) [inline language/Dutch/strings.po:59]: Ah, I know why it was "B":
msgid "Controls Panels"
msgstr "Bedieningspanelen"

Comment (mourix) [reply language/Dutch/strings.po]: Changed back

File: language/Dutch/strings.po (hunk 6 above)
Comment (mourix) [reply language/Dutch/strings.po]: I changed them all!

Comment (happppp) [reply language/Dutch/strings.po]: thanks

**Pull request merged**